app/src/main/java/com/example/cinemapp/ui/main/model/MovieDetails.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.cinemapp.ui.main.model
+
+data class MovieDetails(
+    val id: Int,
+    val backdropPath: String = "",
+    val genres: List<Genre> = emptyList(),
+    val overview: String = "",
+    val runtime: Int = 0,
+    val title: String = ""
+)
+
+data class Genre(
+    val id: Int = -1,
```

Comment:
Why do you only initialize int here to -1? 
`MovieDetails`, `MovieCredits` doesn't have it